app/controllers/container_node_controller.rb

```diff
@@ -7,6 +7,10 @@ class ContainerNodeController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def show_list
+    process_show_list(:where_clause => 'container_nodes.deleted_on IS NULL')
```

Comment:
Could you add `active` and `archived` scopes to model and use them here?
Like in https://github.com/ManageIQ/manageiq/pull/15251.
`deleted_on IS NULL` should keep working with switch to soft deletes, but maybe we'll end up adding a boolean `deleted` column which for more efficient query;
in any case, it's better to encapsulate the logic in the model.